CONTRIBUTING.md

```diff
@@ -41,19 +41,21 @@ The `actions/` directory contains composite actions. Refer to the [GitHub docume
 
 4. **Make Changes:** Make your desired changes to the codebase.
 
-5. **Document Changes:** Document your changes in a `README.md` file. Try to follow the examples set forward in other Readmes.
+5. **Write Tests:** Where it makes sense, please add tests. See `actions/techdocs-rewrite-relative-links` as an example.
 
-6. **Commit Changes:** Commit your changes with clear and concise commit messages.
+6. **Document Changes:** Document your changes in a `README.md` file. Try to follow the examples set forward in other Readmes.
 
-7. **Push Changes:** Push your changes to your forked repository.
+7. **Commit Changes:** Commit your changes with clear and concise commit messages.
 
-8. **Submit a Pull Request:** Submit a pull request to the main repository. Ensure that your pull request is detailed and includes information about the changes made.
+8. **Push Changes:** Push your changes to your forked repository.
 
-9. **Check CI:** Check your pull request to verify that any CI processes are passing.
+9. **Submit a Pull Request:** Submit a pull request to the main repository. Ensure that your pull request is detailed and includes information about the changes made.
 
-10. **Review Process:** Your pull request will be reviewed by Grafana's platform-productivity squad. Make any requested changes or address any feedback during this process.
+10. **Check CI:** Check your pull request to verify that any CI processes are passing.
 
-11. **Contributor License Agreement (CLA):** Before we can accept your pull request, you need to sign [Grafana's CLA](https://grafana.com/docs/grafana/latest/developers/cla/). If you haven't, our CLA assistant prompts you to when you create your pull request.
+11. **Review Process:** Your pull request will be reviewed by Grafana's platform-productivity squad. Make any requested changes or address any feedback during this process.
+
+12. **Contributor License Agreement (CLA):** Before we can accept your pull request, you need to sign [Grafana's CLA](https://grafana.com/docs/grafana/latest/developers/cla/). If you haven't, our CLA assistant prompts you to when you create your pull request.
```

Comment:
nit: I don't think it's our CLA assistant (https://grafana.com/docs/writers-toolkit/review/cla-assistant/)

Reply:
I don't think you need both links, I only included it as a citation that it's not "our CLA assistant". It seems like we have our own CLA but not our own CLA assistant if I'm understanding correctly.

Reply:
Oh, I see what you're saying. I didn't catch that.

I think I'm going to leave it as is for the time being because it matches what's in the `grafana/grafana` contributor's guide, but we can revisit in a future PR.

Thanks for taking the time to review and clarify!